cmd/backup-manager/app/export/export.go

```diff
@@ -115,7 +117,7 @@ func getBackupSize(backupPath string, opts []string) (int64, error) {
 	if exist := backupUtil.IsFileExist(backupPath); !exist {
 		return size, fmt.Errorf("file %s does not exist or is not regular file", backupPath)
 	}
-	args := backupUtil.ConstructArgs(constants.RcloneConfigArg, opts, "ls", backupPath, "")
+	args := backupUtil.ConstructArgs(constants.RcloneConfigArg, nil, "ls", backupPath, "", false)
```

Comment:
Are you sure that no need to add `opts` for `ls`?
Did you test the case where `AWS-KMS` is enabled for S3?

Reply:
In here, `rclone ls` is just used to get the size of local backup archive, just like:
```
$ rclone ls ${archive}
1256 ${archive}
```
So `rclone ls` do not need options about aws, and do not need any options about logging, otherwise, the first word of output may be logging info rather than size number, strToInt will throw a error.

Reply:
Did you test the case where AWS-KMS is enabled for S3?

Reply:
Test done!